Amplify/Categories/DataStore/Query/QuerySortInput.swift

```diff
@@ -0,0 +1,44 @@
+//
+// Copyright 2018-2020 Amazon.com,
+// Inc. or its affiliates. All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+
+/// A simple enum that holding a sorting direction for a field that can be applied to queries.
+public enum QuerySortBy {
+    case ascending(CodingKey)
+    case descending(CodingKey)
+}
+
+public struct QuerySortInput {
+    public let inputs: [QuerySortBy]
+
+    public init(_ inputs: [QuerySortBy]) {
+        self.inputs = inputs
+    }
+
+    /// Creates a `QuerySortInput` in an expressive way, enabling a short
+    /// and developer friendly access to an instance of `QuerySortInput`.
+    ///
+    /// - Parameters:
+    ///   - inputs: a variadic parameters that take uncertain number of `QuerySortBy`
+    /// - Returns: a new instance of `QuerySortInput`
+    public static func by(_ inputs: QuerySortBy...) -> QuerySortInput {
+        return self.init(inputs)
+    }
+
+    /// Unlike `by` above take variadic parameter, `ascending` takes only one `CodingKey`
```

Comment:
No need to compare this to other methods unless it's meaningful to understand the method at the use site. In this case, simply describing it is sufficient: 
```swift
/// Returns an ascending sort specifier for `field`
```
or similar

Reply:
Updated. Also did the same for `descending` one